app/controllers/errors_controller.rb

```diff
@@ -30,6 +30,6 @@ def unprocessable_entity
 private
 
   def set_format
-    request.format = :json if request.original_fullpath.start_with?("/api/") && request.format.html?
+    request.format = :json if request.original_fullpath =~ /^\/api\/v\d+/ && request.format.html?
```

Comment:
This would negatively impact APIs not with a version number (ie qualifications api)

How about just overriding this method to be empty in the guidance controller?